include/ttmlir/Target/Utils/MLIRToFlatbuffer.h

```diff
@@ -83,29 +107,21 @@ inline ::tt::target::DataType toFlatbuffer(FlatbufferObjectCache &,
   }
 }
 
-inline ::tt::target::MemorySpace toFlatbuffer(FlatbufferObjectCache &,
-                                              MemorySpace memspace) {
-  switch (memspace) {
-  case MemorySpace::System:
-    return ::tt::target::MemorySpace::System;
-  case MemorySpace::SystemMMIO:
-    return ::tt::target::MemorySpace::SystemMMIO;
-  case MemorySpace::DeviceDRAM:
-    return ::tt::target::MemorySpace::DeviceDRAM;
-  case MemorySpace::DeviceL1:
-    return ::tt::target::MemorySpace::DeviceL1;
-  }
```

Comment:
Did MemorySpace and Arch need to move?

Reply:
My preference would be to not re-order functions (unless needed) so that the diff here on this commit is smaller and less confusing.